FrameworkSystem/DB/ProxyDB.py

```diff
@@ -370,17 +401,17 @@ def storeProxy(self, userDN, userGroup, chain):
 
     try:
       sUserDN = self._escapeString(userDN)['Value']
-      sUserGroup = self._escapeString(userGroup)['Value']
+      sTable = 'ProxyDB_CleanProxies'
```

Comment:
Again, you seem to completely remove the usecase of DIRAC Group

Reply:
yes, because with new logic proxies uploaded without group

Reply:
See previous comment